src/app/features/categories/categories.component.ts

```diff
@@ -54,6 +54,8 @@ export class CategoriesComponent {
     inputLabelText: string,
     buttonText: string,
     isAdmin: boolean,
+    isEdit = false,
+    editId?: number,
```

Comment:
ta metoda openDialog wyrasta na jakiegos potworka z tymi roznymi parametrami. Przeróbmy to na obiekt konfiguracyjny, będzie czytelniejszy podczas samego wywoływania metody